frontend/src/app/signUp/IdentityVerification/QuestionsFormContainer.tsx

```diff
@@ -95,6 +100,8 @@ const QuestionsFormContainer = ({ personalDetails, orgExternalId }: Props) => {
         questionSet={questionSet}
         saving={false}
         onSubmit={onSubmit}
+        onFail={() => setIdentificationVerified(false)}
```

Comment:
I think we may need to make a call to the backend onFail as well, since we want to send emails to both the user and the support team.

Reply:
Hmm, trying to figure out the best way to do this. We don't have an endpoint to explicitly fail a user out of this process... I suppose we could just submit the answers in whatever state they are in? That would trigger the failure emails, right?

Reply:
Yeah, I was thinking through the same thing. I think submitting the answers at whatever state they're in is good enough, because then we'll also get a record of it (even though the record will almost certainly just say "session expired"). That would trigger all our failure emails and appropriate logging. Will the frontend/backend accept an incomplete list of answers though?

Reply:
Sounds good! Experian should just immediately fail on session timeout anyways, so hopefully our backend won't care if there's an empty array

Reply:
Looks like it works on dev!!